main/ertp/ERTP-Guide/PursesAndPayments.md

```diff
@@ -0,0 +1,269 @@
+# Purses and Payments
+
+You can store `amount` objects in two kinds of objects:
+- **[`purse`](https://agoric.com/documentation/glossary/#purse)**: Hold
+  an `amount` of same branded digital assets until you withdraw part or
+  all of them into a `payment` for use.
+- **[`payment`](https://agoric.com/documentation/glossary/#purse)**:
+  Hold an `amount` of digital assets to send to another party. Usually either
+  a freshly created `amount` or one taken from a `purse`
+
+Assets in purses and payments do not have to be currency-like, but can
+be anything you can construct a digital rights `amount` object for;
+swords to use in a game, rights to use a particular contract, theater
+tickets, etc.
+
+Each non-empty `purse` and `payment` object contains exactly one
+`amount` object. While you may deposit or withdraw an `amount` to a
+`purse`, it is repsectively added to the `purse` `amount`or split off into a `payment`
+object's `amount`. The same is true for adding to or splitting a
+`payment` object's `amount`.  Both a `purse` and a `payment` can only
+have an amount of a single `brand`.
+
+With the exception of `mint` objects creating entirely new `payment`
+objects with newly created `amount` objects, a `payment` object is
+created by making a withdrawal on an `amount` from a `purse`. Note
+that the `brand` of the new `payment` must be the same as the
+`purse`'s associated `brand`.
+
+Note that you don't transfer directly from `purse`s to
+another `purse`. Instead, to send and receive assets in ERTP you do these steps:
```

Comment:
Should we talk about ERTP in the context of our larger system? I think so. In that case, we should talk about depositFacets and how to use them here. I can help with that if you have questions.